objects/o_main_controller/Create_0.gml

```diff
@@ -1,8 +1,9 @@
 /// @description Initialize some variables
 
-hp = 3;
-max_hp = 3;
-sapphires = 0;
+max_hp = 100;
+max_charge = 100;
+player_hp = [max_hp, max_hp];

```

Comment:
Normally I'd keep this information with the o_player objects; I'm putting it here so that it remains between rooms.  If there's a better way to do this I'm totally for it.